config/webpack.config.js

```diff
@@ -219,7 +219,7 @@ module.exports = function (env) {
 				ext => useTypeScript || !ext.includes('ts')
 			),
 			// Allows us to specify paths to check for module resolving.
-			modules: [path.resolve('./node_modules'), 'node_modules'],
+			modules: [path.resolve('./node_modules'), 'node_modules', ...app.webpackExtraResolvePaths],
```

Comment:
dear @ericblade ,
I think inputting only one path(./src) that uses the root directory would be more simple, less confusing to use.
(CRA does, like https://create-react-app.dev/docs/importing-a-component/#absolute-imports)

Would you please share the reason that needs to use `...app.webpackExtraResolvePaths(multiple paths)`?

Reply:
Mostly because the option for webpack's resolve.modules is also an array.  To solve for my specific use case, adding just a single ./src is sufficient, however I could imagine someone might want ['./src', './resources'] or ['./src/common', './src/features'] or ['./src/components', './src/hooks', './src/store' ] something along those lines.  

I was thinking of more broad use cases than what I'm personally using.  Although i think i might make mine ['./src', './resources'] now, that would be useful.

Reply:
We agree, submit this line.

But these paths should be prefixed with the app's path.
Please check this PR(https://github.com/enactjs/dev-utils/pull/85)

Reply:
on checking

Reply:
Thank you for the explanation. 
We think it's not safe to change the default behavior of the current resolving paths rules so we've decided to add this feature optionally via "enact" metadata.
I have some comments.

1. How about renaming "webpackExtraResolvePaths" to "additionalModulePaths"?
2. To broaden the acceptable type of this option, we think we could get a string or an Array. So someone can use "src" and others can use ["src", "resources"]. 
So how about the below changes?
```
const getAdditionalModulePaths = (paths) => {
	if (!paths) return [];
	return Array.isArray(paths) ? paths : [paths];
};

...

// Allows us to specify paths to check for module resolving.
modules: [
	path.resolve('./node_modules'),
	'node_modules',
	...getAdditionalModulePaths(app.additionalModulePaths)
],
```

Reply:
sure, that sounds great.  I'll try to update this soon to include.  Thanks for the feedback!

Reply:
Hi @ericblade ,
Would you please let us know when you could update this PR? Also, could you change the branch name starts with `feature'/` and targetting to `develop`?
If it'll take some time, would you mind if we make another PR?

Reply:
@ericblade, I'm sorry but we have a plan to deliver this feature soon so we'll proceed to add this with another PR.

Reply:
Apologies for not getting to it, things are busy!  

Thanks for including the feature!